caesars-palace/blocks/quick-facts-hours/quick-facts-hours.js

```diff
@@ -0,0 +1,393 @@
+        modalOverlay.classList.remove('quick-facts-modal-overlay-hidden');
+      };
+      statusDiv.addEventListener('click', allHoursViewHandler, { passive: true });
+    }
+  } else {
+    const printedSchedule = new Array(7);
+    const productOpenSchedule = {
+      Sunday: {
+        opens: [],
+      },
+      Monday: {
+        opens: [],
+      },
+      Tuesday: {
+        opens: [],
+      },
+      Wednesday: {
+        opens: [],
+      },
+      Thursday: {
+        opens: [],
+      },
+      Friday: {
+        opens: [],
+      },
+      Saturday: {
+        opens: [],
+      },
+    };
+
+    [...block.children].forEach((row, index) => {
+      printedSchedule[index] = {
+        day: row.children[0].innerText,
+        hours: row.children[1].innerText,
+      };
+      updateOpeningSchedule(
+        productOpenSchedule,
+        row.children[0].innerText,
+        row.children[1].innerText,
+      );
+      row.remove();
+    });
+
+    const modalOverlay = document.createElement('div');
+    modalOverlay.classList.add('quickfacts-modal-overlay');
+    modalOverlay.classList.add('quick-facts-modal-overlay-hidden');
+    const modalDiv = buildHoursModal(printedSchedule, modalOverlay);
+
+    const dateToCheck = new Date();
+    const isOpen = isVentureOpen(productOpenSchedule, dateToCheck);
+    let openingStatusText;
+    let nextStatusChangeTime;
+    let nextStatusChangeTimeText;
+    let statusIconClass;
+    if (isOpen) {
+      openingStatusText = OPEN_TXT;
+      statusIconClass = 'status-open';
+      nextStatusChangeTime = getNextClosing(productOpenSchedule, dateToCheck) || '';
+
+      if (nextStatusChangeTime) {
+        let { minutes } = nextStatusChangeTime;
+        if (minutes < 10) {
+          minutes = `0${minutes}`;
+        }
+        nextStatusChangeTimeText = `${NEXT_CLOSE_TXT} ${nextStatusChangeTime.hours % 12}:${minutes} ${nextStatusChangeTime.halfdayMarker}`;
+      }
+    } else {
+      openingStatusText = CLOSED_TXT;
+      statusIconClass = 'status-closed';
+      nextStatusChangeTime = getNextOpening(productOpenSchedule, dateToCheck) || '';
+
+      if (nextStatusChangeTime) {
+        let { minutes } = nextStatusChangeTime;
+        if (minutes < 10) {
+          minutes = `0${minutes}`;
+        }
+        nextStatusChangeTimeText = `${NEXT_OPEN_TXT} ${nextStatusChangeTime.hours % 12}:${minutes} ${nextStatusChangeTime.halfdayMarker} ${nextStatusChangeTime.day}`;
+      }
+    }
+
+    const statusDiv = document.createElement('div');
+    const statusIconNode = document.createElement('span');
+    const statusTextNode = document.createElement('span');
+    const nextStatusChangeNode = document.createElement('span');
+    statusTextNode.innerText = openingStatusText;
+    nextStatusChangeNode.innerText = nextStatusChangeTimeText || '';
+    const allHours = document.createElement('a');
+    const allHoursDecoration = document.createElement('div');
+    allHours.href = '#';
```

Comment:
Adding a `#` to the href will cause the page to scroll back to the top. `href` isn't needed so you should just be able to omit it.

Reply:
Also, when you are creating an element with many attributes the `createTag` helper function is useful since you can pass an object that includes all your desired attribute values.